demisto_sdk/commands/common/tests/docker_test.py

```diff
@@ -266,11 +270,26 @@ def test_is_docker_image_latest_tag_without_tag(self):
         docker_image_validator.is_latest_tag = True
         docker_image_validator.docker_image_tag = '1.0.2'
         docker_image_validator.is_valid = True
+        docker_image_validator.yml_docker_image = True
 
         assert docker_image_validator.is_docker_image_latest_tag() is False
         assert docker_image_validator.is_latest_tag is False
         assert docker_image_validator.is_docker_image_valid() is False
 
+    @pytest.mark.parametrize('code_type, expected', [('javascript', True), ('python', False)])
+    def test_no_dockerimage_in_yml_file(self, code_type, expected):
+        docker_image_validator = mock_docker_image_validator()
+
+        docker_image_validator.is_valid = True
+        docker_image_validator.is_latest_tag = True
+        docker_image_validator.yml_docker_image = False
```

Comment:
This is not a possible value according to the code, it is better to mock a possible value.